### ISSUE TITLE
Only superuser can edit a app PlaceholderField

### ISSUE THREAD

**Issue by jedie** [divio/django-cms#6153]
I try to edit a placeholder field from a cms app in frontend mode. e.g.: `/admin/cms/page/edit-plugin/<pk>/`

A superuser can edit. But a active, staff but non-superuser with all existing permissions get a permission error.

I figure out that the deny comes from `cms.models.placeholdermodel.Placeholder`:
```
class Placeholder(...):
    ...
    def has_change_permission(self, user):
        ...
        attached_models = self._get_attached_models()
        if not attached_models:
            return False
        ...
```
The `attached_models` list is empty.

The code path is like:
```
File ".../python3.5/site-packages/cms/admin/placeholderadmin.py" in edit_plugin
  638.         if not self.has_change_plugin_permission(request, obj):
File ".../python3.5/site-packages/cms/admin/placeholderadmin.py" in has_change_plugin_permission
  251.         return placeholder.has_change_plugin_permission(request.user, plugin)
File ".../python3.5/site-packages/cms/models/placeholdermodel.py" in has_change_plugin_permission
  170.         if not self.has_change_permission(user):
File ".../python3.5/site-packages/cms/models/placeholdermodel.py" in has_change_permission
  133.             return False
````
(the last `133` line is the result of `if not attached_models:`)

Any idea?

### Environment

* Python version: 3.5.2
* Django version: 1.8.18
* django CMS version: 3.4.5

**Comment by czpython** [comment]
@jedie You need to subclass the `PlaceholderAdminMixin` on the admin class of the model with `PlaceholderField`.

Closing as duplicate of https://github.com/divio/django-cms/issues/5410

**Comment by jedie** [comment]
> You need to subclass the PlaceholderAdminMixin on the admin class of the model with PlaceholderField.

Cool! That's it. Thank you!

**Comment by openHBP** [comment]
I see this issue is closed and is the same as #5178 and #5410 but I miss something...

In my `msg.models.py`, I subclass the PlaceholderAdminMixin like this:

```python
from django.db import models
from cms.models.fields import PlaceholderField
from cms.admin.placeholderadmin import PlaceholderAdminMixin

class MsgDet(PlaceholderAdminMixin, models.Model):
    title = models.CharField(max_length=255)
    short_text = PlaceholderField('msgdet_slot')
```
I've also tried:
```python
from django.db import models
from cms.models.fields import PlaceholderField
from cms.admin.placeholderadmin import PlaceholderAdminMixin

class MyPlaceholderField(PlaceholderAdminMixin, PlaceholderField):
    pass

class MsgDet(models.Model):
    title = models.CharField(max_length=255)
    short_text = MyPlaceholderField('msgdet_slot')
```

None of them are permitting access to my user on the placeholder slot defined as PlaceholderField.

- I've done `makemigration` and `migrate` on my model.
- I've granted `all permissions` to my user.

I always get the same message after double clicking on the `rendered PlaceholderField` on my html page: _"You don't have permission to modify this plugin"_.
What I'm missing here???

I'm with **django 3.0** and **django-cms 3.7.4**
